clang/test/CodeGen/fake-use-small-aggs.c

```diff
@@ -0,0 +1,24 @@
+// RUN: %clang_cc1 %s -O2 -emit-llvm -fextend-lifetimes -disable-llvm-passes -o - | FileCheck %s
+// Check that we generate a fake_use call for small aggregate types.
```

Comment:
So, aggregate types that get splatted into single integers? It's worth identifying that in fake-use-scalar.c just in case someone tries to add test coverage there IMHO.